pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/BaseKitTestUtil.java

```diff
@@ -0,0 +1,264 @@
+package org.broadinstitute.dsm.kits;
```

Comment:
This class consists of shared functionality that used to be in the TestKitUtil and JuniperTestSetupUtil.
In the new structure this class contains what is a standard kit's journey and specific kit setup that belongs to only one of Juniper or Pepper is now in JuniperKitUtil and PepperKitUtil